config/nginx/conf.d/cantusdb.conf

```diff
@@ -16,8 +16,8 @@ server {
 
     listen 443 default_server http2 ssl;
 	
-    ssl_certificate /etc/nginx/ssl/live/cantusdatabase.org/fullchain.pem;
-    ssl_certificate_key /etc/nginx/ssl/live/cantusdatabase.org/privkey.pem;
+    ssl_certificate /etc/nginx/ssl/live/certificates/staging.cantusdatabase.org.crt;
+    ssl_certificate_key /etc/nginx/ssl/live/certificates/staging.cantusdatabase.org.key;
```

Comment:
We should have these set up to work on Production - if a developer misses a step and thus takes down a server, we'd rather that server be Staging than Production.

Reply:
Note that Production has two addresses - `cantusdatabase.org` and `www.cantusdatabase.org`. Do we know how this will work - are there going to be two separate certificates and two separate keys in two separate directories? Are we going to be able to configure nginx to handle this situation?

Reply:
Good point...and good question. Let me look into it.

Reply:
Ok, based on what I am seeing a few places (incl. https://stackoverflow.com/questions/50496214/how-can-i-generate-a-letsencrypt-cert-for-multiple-domain-variants and https://github.com/go-acme/lego/issues/228) it looks to me like we should expect certificates on production to be stored at `/etc/nginx/ssl/live/certificates/cantusdatabase.org`...I'll make that change here.